submissions/Jemikalajah-Philip/php/create.php

```diff
@@ -0,0 +1,56 @@
+<?php
+if (isset($_POST['submit'])){
+    require "config.php";
+    require "helper.php";
+    try {
+        $connection = new PDO ($dsn, $username, $password, $options);
+        $newProduct = array(
+            "itemname" => $_POST['itemname'],
+            "quantity" => $_POST['quantity'],
+            "manufacturer" => $_POST['manufacturer'],
+            "about" => $_POST['about'],
+            "brand_id" => $_POST['brands'] + 1,
```

Comment:
you are adding 1 here because you did not fetch the brand ids from the database in your fetch.php file. You should fetch the brand ids and brand names and create an associative array that maps ids to name and use that to build the <select> tag in your create.php file. That way, you don't need to add +1 anymore.

Think of it this way, what if brand item 3 is removed from your database. what effect will this your method of adding 1 have? problems, i suppose?